lib/currency-locales/locales/tr.yml

```
@@ -1,9 +1,10 @@
+---
 tr:
   number:
     currency:
       format:
         delimiter: "."
-        format: "%u%n"
+        format: "%n %u"
```

Meaningful change. Adding a space

---

```suggestion
        format: "%u%n"
```

based on https://github.com/twitter/twitter-cldr-rb/blob/2b65e4b494924f80a8d15fc0fd2efd0cd8090ba0/resources/locales/tr/numbers.yml#L43